benchmark/atoi.cpp

```diff
@@ -326,7 +325,7 @@ std::size_t c_strLength_natural(const char *s) {
     auto bytes = adjustMisalignmentFor_strlen(initialBytes, misalignment);
     for(;;) {
         auto nulls = zoo::swar::equals(bytes, S{0});
-        if(nulls) { // there is a null!
```

Comment:
Definitively not.
`equals` returns a BooleanSWAR.  You've something broken in your side.